### ISSUE TITLE
Condition is not getting effected when defined in conjunction collection

### ISSUE THREAD

**Issue by NigmaX** [directus/directus#14553]
### Describe the Bug

when trying to define a condition on a field in the conjunction collection, where the field is a relational field. the condition is not applied.

### To Reproduce

- set up 2 collections
- connect a field from the first to the second
- make a condition from the conjunction collection 

expected: condition kicks in
actual: the condition is not working for the junction field

https://user-images.githubusercontent.com/8597752/180250069-b57c443d-cd8d-4eea-b4f0-60eb0d9497fa.mp4

Thanks!

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

16

### What database are you using?

mysql

### What browser are you using?

chrome

### How are you deploying Directus?

local

**Comment by NigmaX** [comment]
@br41nslug just wanna mention that I actually used the code from the fix-v-form branch, you're working on :)

**Comment by br41nslug** [comment]
When testing this issue you mean @NigmaX ? If so then i'll reopen the ticket and have a second look 😄

**Comment by NigmaX** [comment]
@br41nslug yes, but looks like it actually did get resolved on 9.14.3. so it can stay closed :) 
thanks!
